### PR TITLE
trackupstream: add Aodh and Gnocchi.

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

**Comment by toabctl** [comment]
nit: commit msg: it's ```trackupstream``` not ```trackup-upstream```

**Comment by toabctl** [comment]
@aplanas you also need to add it to the combination-filter. Next to where i.e. ```python-networking-cisco``` is.

**Comment by aplanas** [comment]
* Fix name in commit message
* Add packages in the filter (for Liberty)
* Fix space in filter

File: scripts/jenkins/jobs-obs/openstack-trackupstream.yaml [hunk 1]
```diff
@@ -81,7 +85,12 @@
             - cloud-trackupstream
     execution-strategy:
       combination-filter: |
-        ! ( [ "Cloud:OpenStack:Liberty:Staging", "Cloud:OpenStack:Kilo:Staging", "Cloud:OpenStack:Juno:Staging"].contains(project) &&
+        ! ( [ "Cloud:OpenStack:Liberty:Staging" ].contains(project) &&
```
Comment (toabctl) [inline scripts/jenkins/jobs-obs/openstack-trackupstream.yaml]: not needed

File: scripts/jenkins/jobs-obs/openstack-trackupstream.yaml [hunk 2]
```diff
@@ -81,7 +85,12 @@
...
+            [ "openstack-aodh",
+              "openstack-gnocchi",
+	      "python-aodhclient",
+              "python-gnocchiclient" ].contains(component) ||
+            [ "Cloud:OpenStack:Liberty:Staging", "Cloud:OpenStack:Kilo:Staging", "Cloud:OpenStack:Juno:Staging" ].contains(project) &&
```
Comment (toabctl) [inline scripts/jenkins/jobs-obs/openstack-trackupstream.yaml]: not needed

**Comment by toabctl** [comment]
@aplanas you don't need to change so much. The filter is negated. Just add the packages to the list where ```python-networking-cisco``` is in. Then it will be tracked for liberty and master...

**Comment by aplanas** [comment]
@toabctl Oh yes, now I see the big `!`. Thanks!

**Comment by bmwiedemann** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**